engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/DateTest.scala

```diff
@@ -0,0 +1,57 @@
+package org.enso.interpreter.test.semantic
+
+import org.enso.interpreter.test.{InterpreterContext, InterpreterTest}
+
+class DateTest extends InterpreterTest {
+  override def subject: String = "Debug.eval"
+
+  override def specify(implicit
+    interpreterContext: InterpreterContext
+  ): Unit = {
+    "evaluate a date expression" in {
+      val code =
+        s"""from Standard.Builtins import all
+           |
+           |import Standard.Base.Data.Time.Date
+           |
+           |main =
+           |    Debug.eval $rawTQ
+           |        IO.println (Date.new 2022 04 01)
+           |""".stripMargin
+      eval(code)
+      consumeOut shouldEqual List("2022-04-01")
+    }
+
+    "print out java date" in {
+      val code =
+        s"""from Standard.Builtins import all
+           |polyglot java import java.time.LocalDate
+           |
+           |main =
+           |    Debug.eval $rawTQ
+           |        IO.println (LocalDate.of 2022 04 01)
+           |""".stripMargin
+      eval(code)
+      consumeOut shouldEqual List("2022-04-01")
+    }
+
+    "send enso date into java" in {
+      val code =
+        s"""from Standard.Builtins import all
+           |polyglot java import java.time.LocalTime
+           |import Standard.Base.Data.Time.Date
+           |
+           |main =
+           |    Debug.eval $rawTQ
+           |        ensodate = Date.new 2022 04 01
+           |        javatime = LocalTime.of 10 26
+           |        javatimedate = javatime.atDate ensodate
```

Comment:
...at the end I created `LocalTime` instances and called its [atDate](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/LocalTime.html#atDate(java.time.LocalDate)) that one really accepts `LocalDate` parameter.

With the changes in this PR this version of the test passes OK.